File: CLI/GameClasses/AccessoryTruckData.cs
using SiiParser;
using SiiParser.Attributes;

namespace SiiParserSCS.GameClasses
{
    [SiiUnit("accessory_truck_data")]
    public class AccessoryTruckData : GameClass
    {
        [SiiAttribute("info")]
        public string[] Info { get; set; }
    }
}

File: src/game-classes/accessory-truck-data.ts
import SiiUnit from "../decorators/sii-unit";
import SiiAttribute from "../decorators/sii-attribute";
import GameClass from "./game-class";

@SiiUnit("accessory_truck_data")
export class AccessoryTruckData extends GameClass {
    @SiiAttribute("info[]")
    public info: string[];
    public namespace: string;
}